webpack/scenes/RedHatRepositories/index.scss

```diff
@@ -73,3 +73,15 @@
     }
   }
 }
+
+.sticky-pagination {
+  position: sticky;
+  top: 60px;
+  z-index: 999;
+  background-color: white;
+  padding-top: 10px;
+}
+
+.sticky-pagination-grey {
+  background-color: #ededed;
```

Comment:
You should probably use `$color-pf-black-200` here, see a usage [here](https://github.com/Katello/katello/blob/master/webpack/scenes/RedHatRepositories/index.scss#L69).